nsxt/policy_utils.go

```diff
@@ -421,21 +448,47 @@ func nsxtPolicyPathResourceImporter(d *schema.ResourceData, m interface{}) ([]*s
 	return rd, nil
 }
 
-// This importer function accepts policy path only as import ID
-func nsxtPolicyPathOnlyResourceImporter(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
-	return nsxtPolicyPathResourceImporterHelper(d, m)
+func getMultitenancyPathExample(pathExample string) string {
+	return fmt.Sprintf("%s or /orgs/[org]/projects/[project]%s", pathExample, pathExample)
 }
 
-func nsxtVPCPathResourceImporter(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
-	rd, err := nsxtPolicyPathResourceImporterHelper(d, m)
-	if err != nil {
-		return rd, err
+// This importer function accepts policy path or resource ID
+func getFriendlyPolicyPathOrIDResourceImporter(pathExample string) func(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
```

Comment:
I have nothing against friendship, but since the old importers are removed perhaps we can just call this function getPolicyPathOrIdResourceImporter?

Similarly for the one above

Reply:
Makes sense, done